internal/controllers/machine/drain/cache.go

```diff
@@ -32,14 +32,14 @@ const (
 	expirationInterval = 10 * time.Hour
 )
 
-// CacheEntry is an entry of the drain cache. It stores at which time a Machine was drained the last time.
+// CacheEntry is an entry of the requeue cache. It stores at which time a Machine was proceeded the last time.
```

Comment:
Do you mean processed? (Okay saw it below, please fix all occurences of proceeded)